.github/workflows/test-install.yml

```diff
@@ -51,7 +55,8 @@ jobs:
 
     - name: Pip install
       run: |
-        python -m pip --use-feature=2020-resolver install -e .
+        python -m pip --version
+        python -m pip install -e . ${{ matrix.pip-feature-flag }}
```

Comment:
Unrelated to this change, but is there a reason for using `-e` here? Editable installs just add an "egg-link" to the environment, instead of "properly" installing it. Not sure if there might be some issues that we could catch without `-e` that we won't see now.

Reply:
Good point. There is no need for the `-e` here.